pyquil/api/qpu.py

```diff
@@ -247,6 +218,11 @@ def run_and_measure(self, quil_program, qubits, trials=1, needs_compilation=True
         :return: A list of a list of classical registers (each register contains a bit)
         :rtype: list
         """
+        # Developer note: Can't wholesale replace these functions with
```

Comment:
I'm confused - we can't replace them because we _have_ deprecated or _have not_ deprecated the http apis?

Reply:
clarified in 93e68fcd480a5884be1a87d2212661e643d24ce6